src/main/java/gregtech/common/MetaFluids.java

```diff
@@ -130,7 +129,6 @@ public static void init() {
         setDefaultTexture(Materials.LPG, FluidType.NORMAL);
         setDefaultTexture(Materials.SteamCrackedLightFuel, FluidType.NORMAL);
         setDefaultTexture(Materials.SteamCrackedHeavyFuel, FluidType.NORMAL);
-        setDefaultTexture(Materials.UUAmplifier, FluidType.NORMAL);
```

Comment:
Why removing the texture set for UU Amplifier?

Reply:
UU Matter and UU Amplifier were removed in this PR, since we will be replacing them with Positive and Neutral Matter from SoG and Gregicality instead